bots/honkbot.py

```diff
@@ -12,18 +12,21 @@
 class Honkbot(commands.Cog):
     def __init__(self, logger, speedrun_api, google_api, bot=None):
         self.eamuse_maintenance = {
-            "normal": (
+            "daily": (
                 datetime.time(hour=20, tzinfo=pytz.utc),
                 datetime.time(hour=22, tzinfo=pytz.utc),
             ),
             "extended": (
                 datetime.time(hour=17, tzinfo=pytz.utc),
                 datetime.time(hour=22, tzinfo=pytz.utc),
-            ),
-            "us": (
-                datetime.time(hour=12, tzinfo=pytz.utc),
-                datetime.time(hour=17, tzinfo=pytz.utc),
-            ),
+            )
+        }
+
+        self.messages = {
+            'us_games': 'US Servers (DDR White, IIDX Lightning)',
+            'jp_games': 'JP Servers (DDR Gold, IIDX Classic, All Other)',
+            'website_down': 'Login to eAmusement website unavailable during this time.',
+            'card_down': 'eAmusement card cannot be used during this time.'
```

Comment:
If we want to get pedantic, the proper formatting is "e-amusement".

Reply:
I can update this